### PR TITLE
git-branch: add -b explanation

### PR DESCRIPTION
**Pull request by ClarkAllen1556**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by zdroid** [comment]
You can push here, no need to close PR and make new one.

File: pages/common/git-branch.md [hunk 1]
```diff
@@ -30,3 +30,7 @@
 - Delete a local branch (must not have it checked out to do this):
 
 `git branch -d {{branch_name}}`
+
+- Create new branch and then checkout said branch:
```
Comment (zdroid) [inline pages/common/git-branch.md]:
```suggestion
- Create new branch and switch to it:
```

Perhaps this is more clear.

**Comment by principis** [comment]
`git branch -b` does not exist?

**Comment by ivanhercaz** [comment]
Hi @ClarkAllen1556! Thank you for your interest in tldr-pages and your work. However I think you are confusing `git checkout -b` with this. I think this because the specific actions I see in `git branch --help` are:

```
    -a, --all             listar ramas de remote-tracking y locales
    -d, --delete          borrar ramas totalmente fusionadas
    -D                    borrar rama (incluso si no está fusionada)
    -m, --move            mover/renombrar una rama y su reflog
    -M                    mover/renombrar una rama, incluso si el destino existe
    -c, --copy            copiar una rama y su reflog
    -C                    copiar una rama, incluso si el objetivo existe
    -l, --list            listar nombres de ramas
    --create-reflog       crea el reflog de la rama
    --edit-description    edita la descripción de la rama
    -f, --force           fuerza la creación,movimiento/renombrado,borrado
    --merged <confirmar>  muestra solo ramas que han sido fusionadas
    --no-merged <confirmar>
                          muestra solo ramas que no han sido fusionadas
    --column[=<estilo>]   muestra las ramas en columnas
    --sort <clave>        nombre del campo por el cuál ordenar
    --points-at <objeto>  imprimir sólo las ramas del objeto
    -i, --ignore-case     ordenamiento y filtrado son case-insensitive
    --format <formato>    formato para usar para el output
```

**Comment by zdroid** [comment]
Oh, you guys are really right. It's `git checkout -b` and not `git branch -b` the author was thinking of.

**Comment by ClarkAllen1556** [comment]
I totally don't know how I wrote this up with out realizing I writing up the wrong command. It is `git checkout -b`, like top 10 most used git commands. Closing this out; thanks!

**Comment by ClarkAllen1556** [comment]
Wrote up wrong command.

**Pull request closed without merging**